File: 26_Remove_Duplicates_from_Sorted_Array/26_Remove_Duplicates_from_Sorted_Array.c
int removeDuplicates(int* nums, int numsSize){
    int i = 0;
    int j = 0;
    while (i<=j && j<numsSize){
        if (nums[i] == nums[j]){
            j+=1;
        }else{
            nums[i+1] = nums[j];
            i+=1;
            j+=1;
        }
    }
    return i+1;
}

File: 26_Remove_Duplicates_from_Sorted_Array/26_Remove_Duplicates_from_Sorted_Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i,j=0,1
        while i<=j and j<len(nums):
            if nums[i]==nums[j]:
                j+=1
            else:
                nums[i+1]=nums[j]
                i+=1
                j+=1
            print(nums)
        return i+1